internal/v2/domain/policy_mail_text.go

```diff
@@ -0,0 +1,22 @@
+package domain
+
+import "github.com/caos/zitadel/internal/eventstore/models"
+
+type MailText struct {
+	models.ObjectRoot
+
+	State        PolicyState
+	Default      bool
+	MailTextType string
+	Language     string
```

Comment:
up for discussion: language.Tag?

Reply:
I just made exactly the same as we had before. in my opinion mailtexttype should also be a enum